_content/news/_posts/2019-01-25-laarne.markdown

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: "Dames A vs Sparta Laarne"
+date: 2019-01-25 09:00:00
+description: Belangrijke wedstrijd in de strijd om een playoff plaats!
+cover: /news/img/2018-09-22-ploegfoto-damesA.jpg
+permalink: /news/2019-01-25-laarne/
+---
+
+Op zaterdag 2 februari nemen onze Dames A het voor de tweede keer op tegen Sparta Laarne. 
+
+Dit wordt een zeer belangrijke wedstrijd in de strijd om een playoff plaats omdat we momenteel beide met evenveel punten op plaats 8 en 9 staan. De winnaar van deze wedstrijd neemt een belangrijke optie voor een plekje in de playoffs.
```

Comment:
De winnaar van deze wedstrijd neemt een belangrijke optie voor een plekje in de playoffs, want beide ploegen staan met evenveel punten op plaatsen 8 en 9 in de Topdivisie! Het wordt dus een zeer belangrijke wedstrijd.